keps/sig-node/4004-deprecate-kube-proxy-version/kep.yaml

```
@@ -0,0 +1,41 @@
+title: Deprecate status.nodeInfo.kubeProxyVersion field
+kep-number: 4004
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-node
+participating-sigs: []
```

```suggestion
owning-sig: sig-network
participating-sigs:
  - sig-node
```

which implies moving the files to `keps/sig-network` too.

Even though this is about a change to Node, if sig-network is going to be driving this, then we should be the owning-sig.